core/utils/poly-look/src/visualisations/charts/flow-charts/sankeyDiagram/sankeyDiagram.js

```diff
@@ -0,0 +1,203 @@
+import { Chart } from "../../chart";
+
+import * as d3 from "d3";
+import * as d3Sankey from "d3-sankey";
+
+//default values
+const linkSource = ({ source }) => source,
+  linkTarget = ({ target }) => target,
+  linkValue = ({ value }) => value,
+  intern = (value) =>
+    value !== null && typeof value === "object" ? value.valueOf() : value,
+  nodeWidth = 5,
+  nodePadding = 5,
+  linkMixBlendMode = "multiply",
+  linkPath = d3Sankey.sankeyLinkHorizontal(),
+  defaultNodeTitle = (d) => `${d.id}: ${d.value}`,
+  defaultMargin = {
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+  },
+  defaultLinkOpacity = 0.5,
+  defaultNodeLabelTextOpacity = 1,
+  defaultTextColor = "#0F1938",
+  defaultLinkColor = "blue",
+  defaultNodeColor = "black",
+  nodeLabelPadding = 10,
```

Comment:
We might want to have module-level constants for these... Not too sure about it, anyhow. Over all because of the sheer amount of them that's needed...